### PR TITLE
Adding network scenario for Prow

### PR DESCRIPTION
**Pull request by shahsahil264**
@paigerube14 @chaitanyaenr

### CODE REVIEW

**Comment by shahsahil264** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests-413

**Comment by shahsahil264** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests-413

**Comment by shahsahil264** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests-413

**Comment by shahsahil264** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests-413

**Comment by shahsahil264** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests-413

**Comment by shahsahil264** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests-413

**Comment by shahsahil264** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests-413

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests-413

**Comment by shahsahil264** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests-413

**Comment by shahsahil264** [comment]
/pj-rehearse periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests-413

File: ci-operator/step-registry/redhat-chaos/network-chaos/redhat-chaos-network-chaos-workflow.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+      NETWORK_PARAMS: ""
+      WAIT_DURATION: "300"
+      RETRY_WAIT: "60"
+      ENABLE_ALERTS: "True"
+      ALERTS_PATH: "/root/kraken/config/alerts"       
+      CHECK_CRITICAL_ALERTS: "True"
+      TELEMETRY_ENABLED: "True"
+      TELEMETRY_API_URL: "https://ulnmf9xv7j.execute-api.us-west-2.amazonaws.com/production"
+      TELEMETRY_USERNAME: "redhat-chaos"
+      TELEMETRY_RUN_TAG: "prow-pvc-fill"
```
Comment (chaitanyaenr) [inline ci-operator/step-registry/redhat-chaos/network-chaos/redhat-chaos-network-chaos-workflow.yaml:24]: s/prow-pvc-fill/network-chaos

File: ci-operator/step-registry/redhat-chaos/network-chaos/redhat-chaos-network-chaos-ref.yaml [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    - name: CHECK_CRITICAL_ALERTS
+      default: "True"
+    - name: TELEMETRY_ENABLED
+      default: "True"
+    - name: TELEMETRY_API_URL
+      default: "https://ulnmf9xv7j.execute-api.us-west-2.amazonaws.com/production"
+    - name: TELEMETRY_USERNAME
+      default: "redhat-chaos"
+    - name: TELEMETRY_RUN_TAG
+      default: "prow-pvc-fill"
```
Comment (chaitanyaenr) [inline ci-operator/step-registry/redhat-chaos/network-chaos/redhat-chaos-network-chaos-ref.yaml:48]: s/prow-pvc-fill/prow-network-chaos

**Comment by shahsahil264** [comment]
/pj-rehearse pull-ci-redhat-chaos-prow-scripts-main-4.14-nightly-krkn-hub-tests-414

**Comment by shahsahil264** [comment]
/pj-rehearse periodic-ci-redhat-chaos-prow-scripts-main-4.16-nightly-krkn-hub-tests-416

**Comment by shahsahil264** [comment]
/pj-rehearse periodic-ci-redhat-chaos-prow-scripts-main-4.16-nightly-krkn-hub-tests-416

**Comment by shahsahil264** [comment]
/pj-rehearse periodic-ci-redhat-chaos-prow-scripts-main-4.16-nightly-krkn-hub-tests-416

**Comment by shahsahil264** [comment]
/pj-rehearse pull-ci-redhat-chaos-prow-scripts-main-4.15-nightly-krkn-hub-tests-415

File: ci-operator/step-registry/redhat-chaos/network-chaos/redhat-chaos-network-chaos-commands.sh [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+set -o pipefail
+set -x
+cat /etc/os-release
+oc config view
+oc projects
+python3 --version
+pushd /tmp
+
+ls -la /root/kraken
+git clone https://github.com/redhat-chaos/krkn-hub.git
```
Comment (paigerube14) [inline ci-operator/step-registry/redhat-chaos/network-chaos/redhat-chaos-network-chaos-commands.sh:13]: we don't need to re-clone krkn-hub that folder structure/packages should already be there

You should add this to get data into es: https://github.com/openshift/release/blob/26858f402a2b1a3233c1c434ddb4bb9e933b9ffd/ci-operator/step-registry/redhat-chaos/container-scenarios/etcd-hangup/redhat-chaos-container-scenarios-etcd-hangup-commands.sh#L14C1-L19C1

**Comment by shahsahil264** [comment]
/pj-rehearse periodic-ci-redhat-chaos-prow-scripts-main-4.16-nightly-krkn-hub-tests-416

**Comment by shahsahil264** [comment]
/pj-rehearse periodic-ci-redhat-chaos-prow-scripts-main-4.16-nightly-krkn-hub-tests-416

**Comment by shahsahil264** [comment]
/pj-rehearse periodic-ci-redhat-chaos-prow-scripts-main-4.16-nightly-krkn-hub-tests-416

**Comment by shahsahil264** [comment]
/pj-rehearse periodic-ci-redhat-chaos-prow-scripts-main-4.16-nightly-krkn-hub-tests-416

**Comment by shahsahil264** [comment]
/pj-rehearse periodic-ci-redhat-chaos-prow-scripts-main-4.16-nightly-krkn-hub-tests-416

**Comment by shahsahil264** [comment]
/pj-rehearse periodic-ci-redhat-chaos-prow-scripts-main-4.16-nightly-krkn-hub-tests-416

**Comment by shahsahil264** [comment]
/pj-rehearse periodic-ci-redhat-chaos-prow-scripts-main-4.16-nightly-krkn-hub-tests-416

**Comment by shahsahil264** [comment]
/pj-rehearse periodic-ci-redhat-chaos-prow-scripts-main-4.16-nightly-krkn-hub-tests-416

**Comment by shahsahil264** [comment]
/pj-rehearse periodic-ci-redhat-chaos-prow-scripts-main-4.16-nightly-krkn-hub-tests-416

**Comment by shahsahil264** [comment]
/pj-rehearse periodic-ci-redhat-chaos-prow-scripts-main-4.16-nightly-krkn-hub-tests-416

**Comment by shahsahil264** [comment]
/pj-rehearse periodic-ci-redhat-chaos-prow-scripts-main-4.16-nightly-krkn-hub-tests-416

**Comment by shahsahil264** [comment]
/pj-rehearse periodic-ci-redhat-chaos-prow-scripts-main-4.16-nightly-krkn-hub-tests-416

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-redhat-chaos-prow-scripts-main-4.16-nightly-krkn-hub-tests-416

**Comment by shahsahil264** [comment]
/pj-rehearse periodic-ci-redhat-chaos-prow-scripts-main-4.16-nightly-krkn-hub-tests-416

**Comment by shahsahil264** [comment]
/pj-rehearse periodic-ci-redhat-chaos-prow-scripts-main-4.16-nightly-krkn-hub-tests-416

**Comment by shahsahil264** [comment]
/pj-rehearse periodic-ci-redhat-chaos-prow-scripts-main-4.16-nightly-krkn-hub-tests-416

**Comment by shahsahil264** [comment]
/pj-rehearse periodic-ci-redhat-chaos-prow-scripts-main-4.16-nightly-krkn-hub-tests-416

**Comment by shahsahil264** [comment]
/pj-rehearse periodic-ci-redhat-chaos-prow-scripts-main-4.16-nightly-krkn-hub-tests-416

File: ci-operator/step-registry/redhat-chaos/krkn-hub-tests/redhat-chaos-krkn-hub-tests-chain.yaml [hunk 4]
```diff
@@ -11,6 +11,7 @@ chain:
   - name: PROFILE_TYPE
     default: "reporting"
   steps:
+  - ref: redhat-chaos-network-chaos
```
Comment (paigerube14) [inline ci-operator/step-registry/redhat-chaos/krkn-hub-tests/redhat-chaos-krkn-hub-tests-chain.yaml:14]: lets move this test case till after the cluster-density-v2 so we have load on the cluster when performing this scenario

Comment (shahsahil264) [reply ci-operator/step-registry/redhat-chaos/krkn-hub-tests/redhat-chaos-krkn-hub-tests-chain.yaml:14]: Sure!

**Comment by shahsahil264** [comment]
/pj-rehearse periodic-ci-redhat-chaos-prow-scripts-main-4.16-nightly-krkn-hub-tests-416

**Comment by shahsahil264** [comment]
/pj-rehearse periodic-ci-redhat-chaos-prow-scripts-main-4.16-nightly-krkn-hub-tests-416

**Comment by shahsahil264** [comment]
/pj-rehearse periodic-ci-redhat-chaos-prow-scripts-main-4.16-nightly-krkn-hub-tests-416

**Comment by shahsahil264** [comment]
@paigerube14 @chaitanyaenr PTAL, All tests Passed!

https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/openshift_release/44293/rehearse-44293-periodic-ci-redhat-chaos-prow-scripts-main-4.16-nightly-krkn-hub-tests-416/1788612568399482880

**Pull request merged**